ReactWindows/ReactNative/Modules/Camera/CameraRollManager.cs

```diff
@@ -0,0 +1,156 @@
+using Newtonsoft.Json.Linq;
+using ReactNative.Bridge;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Windows.Storage;
+using Windows.Storage.Search;
+using Windows.Storage.FileProperties;
+
+namespace ReactNative.Modules.Camera
+{
+    /// <summary>
+    /// Native module CameraRoll.
+    /// CameraRoll provides access to the local camera roll / gallery.
+    /// </summary>
+    public class CameraRollManager : ReactContextNativeModuleBase
+    {
+        // Supported media file types
+        private static readonly List<string> SupportedVideoFileTypes = new List<string> { ".mp4", ".wmv", ".avi" };
+        private static readonly List<string> SupportedImageFileTypes = new List<string> { ".jpeg", ".jpg", ".png", ".gif", ".bmp" };
+
+        /// <summary>
+        /// Instantiates the <see cref="CameraRollManager"/>. 
+        /// </summary>
+        /// <param name="reactContext">The React context.</param>
+        public CameraRollManager(ReactContext reactContext) : base(reactContext)
+        {
+        }
+
+        /// <summary>
+        /// The name of the native module.
+        /// </summary>
+        public override string Name
+        {
+            get { return "CameraRollManager"; }    
+        }
+
+        /// <summary>
+        /// Saves the photo or video to the camera roll / gallery.
+        /// </summary>
+        [ReactMethod]
+        public async void saveToCameraRoll(JObject options, IPromise promise)
+        {
+            // TODO: tag, type?
+            promise.Reject(new NotImplementedException("CameraRoll.saveToCameraRoll() method not yet implemented."));
+        }
+
+        /// <summary>
+        /// Returns a Promise with photo/video identifier objects from the local camera roll.
+        /// </summary>
+        [ReactMethod]
+        public async void getPhotos(JObject options, IPromise promise)
+        {                
+            try
+            {                 
+                if (options["first"] == null)
+                {
```

Comment:
We have some helpers for JObject if you include `using ReactNative.Common;` at the top of the file. It will import the extension method `JObject.ContainsKey`.